lib/safe_regexp.rb

```diff
@@ -8,7 +8,7 @@ class RegexpTimeout < Timeout::Error
   end
 
   class << self
-    def execute(regex, method, string, timeout: 1, keepalive: 10)
+    def execute(regex, method, string = nil, timeout: 1, keepalive: 10)
```

Comment:
can we change that to `*args` so we support everything ?

Reply:
Good idea, amended